CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - Moved the `instance_info_fails` metric under `get_api_requests`
 - Improved [readme](README.md) and added links to more detailed information,
   now featured in subject-specific docs.
+- PUT requests on /mmds always return 204 on success.
```

Comment:
The changes made after the last release should be added to a new section called 'Unreleased' (checkout [How to Keep a CHANGELOG](https://keepachangelog.com/en/0.3.0/) ):
So it should look something like this:
```markdown
# Changelog

## Unreleased

### Changed
- PUT requests on /mmds always return 204 on success.

## [0.11.0]

### Added
...
```